src/pokemon_debug.c

```diff
@@ -623,6 +656,107 @@ void BattleLoadOpponentMonSpriteGfxCustom(u16 species, bool8 isFemale, bool8 isS
     LoadPalette(gDecompressionBuffer, 0x80 + battlerId * 16, 0x20);
 }
 
+//Battle background functions
+struct BattleBackground
+{
+    const void *tileset;
+    const void *tilemap;
+    const void *entryTileset;
+    const void *entryTilemap;
+    const void *palette;
+};
+extern const struct BattleBackground sBattleTerrainTable[];
```

Comment:
This should be defined somewhere near the top of the file, maybe below `sBgColor` imo.